compio-fs/src/stdio/unix.rs

```diff
@@ -0,0 +1,111 @@
+use std::{io, mem::ManuallyDrop};
+
+use compio_buf::{BufResult, IoBuf, IoBufMut};
+use compio_driver::{AsRawFd, FromRawFd, RawFd};
+use compio_io::{AsyncRead, AsyncWrite};
+use compio_runtime::TryAsRawFd;
+
+use crate::pipe::{Receiver, Sender};
+
+/// A handle to the standard input stream of a process.
+///
+/// See [`stdin`].
+pub struct Stdin(ManuallyDrop<Receiver>);
+
+impl Stdin {
+    pub(crate) fn new() -> Self {
+        Self(ManuallyDrop::new(unsafe {
```

Comment:
Should we add some unsafe comment to explain why safe to call this unsafe fn here?

Reply:
Added.